docs/intro.md

```diff
@@ -1,22 +1,105 @@
 # Books with Jupyter 2.0
 
-This tool provides a command-line interface and a Python API that lets users
-do the following:
-
-* Write their content in markdown files or Jupyter Notebooks.
-* Include computational elements in any of the above (e.g. executable
-  code cells along with their outputs)
-* Include rich syntax for publication features, such as citations,
-  cross-references, and equations.
-* Using a simple command, convert this content into:
-    * A web-based interactive book
-    * A publication-quality PDF
+Jupyter Book 2.0 allows users to
+
+* write their content in [markdown files](https://myst-parser.readthedocs.io/en/latest/) or [Jupyter](https://jupyter.org/) notebooks,
+* include computational elements (e.g., code cells) in either type,
+* include rich syntax such as citations, cross-references, and numbered equations, and
+* using a simple command, run the embedded code cells, [cache](https://jupyter-cache.readthedocs.io/en/latest/) the outputs and convert this content into:
+    * a web-based interactive book and
+    * a publication-quality PDF.
 
 ```{warning}
-This is an early prototype tool that may evolve quickly. Your feedback is
-very welcome! To give it, please [open an issue](https://github.com/ExecutableBookProject/cli/issues/new)
+This is an early prototype that may evolve quickly. All [feedback](https://jupyter-cache.readthedocs.io/en/latest/) is very welcome!
 ```
 
+
+## A Small Example Project
+
+[Here's](https://executablebookproject.github.io/quantecon-mini-example/docs/index.html)
```

Comment:
In general, it's better to hyper-link multiple words in a phrase so that it's more noticeable, also better to hyper-link the "content" words rather than the verbs, IMO. That way, it's easier to quickly notice the words that stand out and know where it's pointing.

How about we hyperlink "a short example of a web-based book" rather than "Here's"?

Reply:
Done.